cli/code_scanner.py

```diff
@@ -142,10 +175,20 @@ def scan_path(context: click.Context, path):
 
 @click.command()
 @click.argument("ignored_args", nargs=-1, type=click.UNPROCESSED)
+@click.option('--sca-scan',
```

Comment:
instead of adding this flag explicity for each scan move it to `code_scan` func in cycode.py

Reply:
But it isn't relevant to all for example **pre_receive_scan** I amn't sure it's ok to put it there?

Reply:
its optional flag, so we will not use it in the irrelevant scans